tests/browserstack/gez-simple-lookup.test.js

```diff
@@ -29,10 +29,10 @@ describe('gez-simple-lookup.test.js', () => {
     }, 50000000)
   })
 
-  console.info('versionsMobile - ', versionsMobile.map(ver => `${ver.device_browser} ${ver.device} ${ver.os} ${ver.os_version}` ))
+  // console.info('versionsMobile - ', versionsMobile.map(ver => `${ver.device_browser} ${ver.device} ${ver.os} ${ver.os_version}` ))
 
   versionsMobile.forEach(version => {
-    it(`should execute no result (gez) lookup - ${version.name}`, async () => {
+    it.skip(`should execute no result (gez) lookup - ${version.name}`, async () => {

```

Comment:
should this remain skipped?

Reply:
the same - it is only for current mobile testing purposes.
Mobile testing on real mobile devices are rather slow